### DISCUSSION TITLE
Say hello 👋

### DISCUSSION THREAD

**Discussion by i-shivamsoni** [education/hackconviii#8] [General] [answered]
Do we have to say hello at Discord or we can say hello at the live stream's??

**Comment by akritiacharya** [comment]
hello

**Reply by BazilNizam** [reply]
hellooo

**Comment by usamaeltmsah** [comment]
Hello everyone

**Comment by ishsngh** [comment]
Hello guys!

**Comment by elisemoe** [comment, marked as answer]
> Do we have to say hello at Discord or we can say hello at the live stream's??

You can complete the first mission, saying hello 👋, on either platform!

**Comment by adityaoberai** [comment]
hello :)

**Comment by jaypatel15406** [comment]
hello 👋

**Comment by shaksham08** [comment]
Hello

**Comment by mohdsaabith** [comment]
hello

**Comment by sohansputhran** [comment]
Hello :)

**Comment by HrithikSahu** [comment]
Hey guys, Let's Connect and Discuss Tech stuff :)))

**Comment by dipakparmar** [comment]
Hola, Hello, Bonjour 👋🏻

**Comment by arijitdas123student** [comment]
Hello folks ! 👋👋
How are you all doing ?

**Comment by ibnjunaid** [comment]
Hi Everyone

**Comment by arpan45** [comment]
Hello 👋

**Comment by sabasaeed408** [comment]
hello

**Comment by surajsahani** [comment]
hi  @elisemoe  @juanpflores
